src/js/components/Form/index.d.ts

```diff
@@ -10,7 +10,7 @@ export interface FormProps<T> {
   errors?: {};
   infos?: {};
   messages?: { invalid?: string; required?: string };
-  onChange?: (value: T) => void;
+  onChange?: (value: T, options: { touched?: Record<string, boolean> }) => void;
```

Comment:
I had to give this object a name and to be honest, I don't know if 'options' is a good name, but I can't think of a better name.